src/lib.rs

```diff
@@ -81,12 +81,17 @@ pub trait WeightedUtxo {
 /// the Bitcoin Core wallet written in C++.  Therefore, this implementation attempts to return the
 /// same results as one would find if running the Core wallet.
 ///
+/// If the maximum weight is exceeded, then the least valuable inputs are removed.  In so doing,
+/// minimize the number of UTXOs included in the result.
+///
 /// # Parameters
 ///
 /// * target: Target spend `Amount`.
 /// * cost_of_change: The `Amount` needed to produce a change output.
 /// * fee_rate:  Needed to calculate the effective_value of an output.
 /// * long_term_fee_rate: Needed to estimate the future effective_value of an output.
+/// * `max_weight` - the maximum selection weight allowed.  For `TRUC` transactions, use a weight
+///   of 10,000 vB (40,000 WU) for this parameter.
```

Comment:
Note that the limit is 40,000 WU for TRUC transactions that only use confirmed inputs, but TRUC transactions that spend unconfirmed inputs are limited to 4,000 WU by standardness rules. In case your coin selection permits unconfirmed inputs, you will need to ensure that TRUC transactions can only spend confirmed outputs or unconfirmed outputs from TRUC transactions, and that the limit is reduced when unconfirmed outputs are consumed by a transaction.

Additionally, if you allow spending of unconfirmed UTXOs, you must ensure that TRUC outputs are not spend by non-TRUC transactions.

Reply:
Hmm maybe it's best to just leave out the TRUC portion is this doc since there's not a clear default as I first thought.  This is meant as being a brief description of the parameters although I suppose it could be expanded to include more detail of TRUC transactions if needed also.